### PR TITLE
Add buf mod open

### PR DESCRIPTION
**Pull request by bufdev**
Fixes https://github.com/bufbuild/buf/issues/850

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -11,6 +11,7 @@ require (
 	github.com/klauspost/compress v1.14.2
 	github.com/klauspost/pgzip v1.2.5
 	github.com/oklog/ulid/v2 v2.0.2
+	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8
```
Comment (bufdev) [inline go.mod:14]: BSD 2-Clause license, well-maintained, very small

File: private/buf/cmd/buf/command/mod/modopen/modopen.go [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	"github.com/bufbuild/buf/private/buf/bufcli"
+	"github.com/bufbuild/buf/private/bufpkg/bufconfig"
+	"github.com/bufbuild/buf/private/pkg/app/appcmd"
+	"github.com/bufbuild/buf/private/pkg/app/appflag"
+	"github.com/bufbuild/buf/private/pkg/storage/storageos"
+	"github.com/pkg/browser"
+	"github.com/spf13/cobra"
+)
+
+// NewCommand returns a new prune Command.
```
Comment (johanbrandhorst) [inline private/buf/cmd/buf/command/mod/modopen/modopen.go:30]:
```suggestion
// NewCommand returns a new open Command.
```

**Review by johanbrandhorst — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (mfridman) [reply go.mod:14]: Nice one, didn't know about this package.

**Review by mfridman — APPROVED** [review]
Tested this out, works nicely.

**Pull request merged**